tests/performance/framework.go

```diff
@@ -53,12 +56,27 @@ func init() {
 	}
 }
 
-func SIGDescribe(text string, body func()) bool {
-	return Describe("[sig-performance][Serial] "+text, body)
+func SIGDescribe(text string, body ...interface{}) bool {
+	if !isIncludeSerial(body...) {
+		body = append(body, Serial)
+	}
+	return Describe("[sig-performance][Serial] "+text, body...)
+}
+
+func FSIGDescribe(text string, body ...interface{}) bool {
+	if !isIncludeSerial(body...) {
+		body = append(body, Serial)
+	}
+	return FDescribe("[sig-performance][Serial] "+text, body...)
 }
 
-func FSIGDescribe(text string, body func()) bool {
-	return FDescribe("[sig-performance][Serial] "+text, body)
+func isIncludeSerial(body ...interface{}) bool {
+	for _, item := range body {
+		if reflect.TypeOf(item) == serialType && item == Serial {
```

Comment:
Will [this](https://pkg.go.dev/go/types#Identical) work here?

Reply:
yes.

Reply:
no, not really. The types package is not meant to work like that and won't help here.

Reply:
Anyway, this code was removed as result of the previous comment